clusters/app.ci/vault/secret-collection-manager.yaml

```diff
@@ -1,34 +1,18 @@
-apiVersion: image.openshift.io/v1
-kind: ImageStream
+apiVersion: v1
```

Comment:
I accidentally included a WIP version of this in b5defe31109213e375bb8cc6c0626f78515fb7de, hence the diff